bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/i18n/avmfritz.properties

```diff
@@ -200,40 +164,35 @@ channel-type.avmfritz.radiator_mode.state.option.COMFORT = Comfort
 channel-type.avmfritz.radiator_mode.state.option.ECO = Eco
 channel-type.avmfritz.radiator_mode.state.option.BOOST = Boost
 channel-type.avmfritz.radiator_mode.state.option.WINDOW_OPEN = Window Open
-
-channel-type.avmfritz.next_change.label = Next Setpoint Change
-channel-type.avmfritz.next_change.description = Next change of Setpoint Temperature.
-channel-type.avmfritz.next_change.pattern = %1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS
-
-channel-type.avmfritz.next_temp.label = Next Setpoint Temperature
-channel-type.avmfritz.next_temp.description = Next Setpoint Temperature.
-
-channel-type.avmfritz.contact_state.label = Contact State
-channel-type.avmfritz.contact_state.description = Contact state information (OPEN/CLOSED).
-
-thing-type.avmfritz.HAN_FUN_SWITCH.channel.press.label = Button Press
-thing-type.avmfritz.HAN_FUN_SWITCH.channel.press.description = Triggers PRESSED when a button was pressed.
-
-thing-type.avmfritz.FRITZ_DECT_400.channel.press.label = Button Press
-thing-type.avmfritz.FRITZ_DECT_400.channel.press.description = Triggers SHORT_PRESSED or LONG_PRESSED when a button was pressed.
-
 channel-type.avmfritz.rollershutter.label = Rollershutter Control
 channel-type.avmfritz.rollershutter.description = Controls the rollershutter and states its opening level in percent
-
-channel-type.avmfritz.last_change.label = Last Change
-channel-type.avmfritz.last_change.description = States the last time the button was pressed.
-channel-type.avmfritz.last_change.pattern = %1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS
+channel-type.avmfritz.set_temp.label = Setpoint Temperature
+channel-type.avmfritz.set_temp.description = Thermostat Setpoint temperature.
+channel-type.avmfritz.temperature.label = Current Temperature
+channel-type.avmfritz.temperature.description = Current measured temperature.
+channel-type.avmfritz.voltage.label = Voltage
+channel-type.avmfritz.voltage.description = Current voltage.
 
 # channel types config
+
 channel-type.config.avmfritz.temperature.offset.label = Temperature Offset
 channel-type.config.avmfritz.temperature.offset.description = Current temperature offset (in °C).
 
+# channel types
+
+channel-type.avmfritz.last_change.pattern = %1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS
+channel-type.avmfritz.next_change.pattern = %1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS
+thing-type.avmfritz.FRITZ_DECT_400.channel.press.label = Button Press
+thing-type.avmfritz.FRITZ_DECT_400.channel.press.description = Triggers SHORT_PRESSED or LONG_PRESSED when a button was pressed.
+thing-type.avmfritz.HAN_FUN_SWITCH.channel.press.label = Button Press
+thing-type.avmfritz.HAN_FUN_SWITCH.channel.press.description = Triggers PRESSED when a button was pressed.
```

Comment:
Are you sure those patterns even work?
I would think those patterns only work on state so the key ends with `.state.pattern`
The plugin will already [add patterns](https://github.com/wborn/openhab-core/blob/399a15210ae8109f0ecc4a464c0ed3486a14ada6/tools/i18n-plugin/src/main/java/org/openhab/core/tools/i18n/plugin/XmlToTranslationsConverter.java#L284-L288) for state that has these and which are time patterns.
It would be nice if OHC would use a locale specific date pattern when the state has no pattern.

Reply:
Yes, you're right. Good catch. I cleaned-up the mess in https://github.com/openhab/openhab-addons/pull/11545.